app/views/admin/products_v3/_sort.html.haml

```diff
@@ -6,4 +6,4 @@
         = t(".pagination.clear_search")  
   %form.with-dropdown
     = t(".pagination.per_page.show")
-    = select_tag :per_page, options_for_select([15, 25, 50, 100].collect{|i| [t('.pagination.per_page.per_page', num: i), i]}, pagy.items), data: { reflex: "change->products#change_per_page" }
+    = select_tag :per_page, options_for_select([15, 25, 50, 100].collect{|i| [t('.pagination.per_page.per_page', num: i), i]}, pagy.items), class: "no-input", data: { reflex: "change->products#change_per_page", controller: "tom-select", "tom-select-options-value": '{ "plugins": [] }'}
```

Comment:
Adding `tom-select` stimulus controller seems unrelated to this commit, why is it needed ?

Reply:
Sorry, the commit message should have explained better. 
This issue introduces a new design for single-select dropdowns.
We already had some single-select tom-select dropdowns in the admin interface, so the updated design applies to them. 
Tom-select is also added to this select, so we can apply the new design to it as required by the issue. Hope that makes sense.